docs/SPEC.md

```diff
@@ -520,6 +520,12 @@ If type inference determines all the properties on a record it is said to be bou
 Not all keys may be known on the type of a record in which case the record is said to be unbounded.
 An unbounded record may contain any property in addition to the properties it is known to contain.
 
+##### Dictionary types
+
+A _dictionary type_ is a collection that associates keys to values.
+Keys must be comparable and of the same type.
+Values must also be of the same type.
+
```

Comment:
Should we explicitly mention here that dictionary keys can be dynamic, i.e. any expression?

Reply:
Yes I mentioned that in the dictionary literal section